.eslintrc.js

```diff
@@ -31,6 +30,11 @@ module.exports = {
     '@typescript-eslint/no-explicit-any': 'off',
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/no-angle-bracket-type-assertion': 'off',
+    // '@typescript-eslint/consistent-type-assertions': [
```

Comment:
Was this intended to be commented out?

Reply:
Unfortunately yes as this is currently breaking the codacy tests. I will need to uncomment these rules as soon as they upgrade to ESLint v6

Reply:
Ah yeah - I read the info in comment after the fact, but thank you for clarifying!